defaults/main.yml

```diff
@@ -1,2 +1,16 @@
 ---
-# defaults file for frr
\ No newline at end of file
+# defaults file for frr
+
+frr_version: "{{ version | default('6.0.2') }}"
+frr_create_service_account: "{{ create_service_account | default(False) }}"
+frr_remove_service_account: "{{ remove_service_account | default(False) }}"
+frr_dependent_role_check: "{{ dependent_role_check | default(True) }}"
+
+temp_daemons_list_1: "{{ enable_daemons | default([]) }}"
+frr_enable_daemons: "{{ temp_daemons_list_1 + ['zebra'] }}"
+
+frr_packages:
+  frr_6.0.2_CentOS_7: https://github.com/FRRouting/frr/releases/download/frr-6.0.2/frr-6.0.2-01.el7.centos.x86_64.rpm
+  frr_6.0.2_CentOS_6: https://github.com/FRRouting/frr/releases/download/frr-6.0.2/frr-6.0.2-01.el6.x86_64.rpm
+  frr_5.0.2_CentOS_7: https://github.com/FRRouting/frr/releases/download/frr-5.0.2/frr-5.0.2-001.el7.centos.x86_64.rpm
+  frr_5.0.2_CentOS_6: https://github.com/FRRouting/frr/releases/download/frr-5.0.2/frr-5.0.2-001.el6.x86_64.rpm
```

Comment:
Why are the versions hard coded?

Reply:
If custom_package is not provided, based on `ansible_distribution_version` you can set frr version and then concat the variable to url and pass that to `yum` module. If the versions do not fall into 5.0.2 and 6.0.2 criteria, Fail?

Reply:
@trishnaguha Custom package is optional. When provided, it gains precedence over selecting just the version of FRR to install. The main aim of having that option was to allow the users to install any older version of FRR. We by default, provide the last 2 releases, since it's not feasible for us to hardcode all the release package URLs multiplied by all the CentOS version supported.

Reply:
The URLs are hardcoded because the releases are not yet available using yum. This is how FRR has to be installed manually(https://github.com/FRRouting/frr/releases). The project forked after RHEL 7 released. Probably, the next RHEL release would have it.